lib/manifold.ex

```diff
@@ -83,7 +89,7 @@ defmodule Manifold do
       :offload ->
         # To maintain linearizability guaranteed by send/2, we have to send
         # it to the sender process, even for a single receiving pid.
-        Sender.send(current_sender(), current_partitioner(), [pid], message)
+        Sender.send(current_sender(), current_partitioner(), [pid], message, :etf)
```

Comment:
`options[:pack_mode]`?

Reply:
In this case we are only sending to a single pid, so we don't get any benefit from doing the term_to_binary ourself, so I just skipped it.  This code was skipping it for the single pid case, which seemed reasonable and, I thought, purposeful.

Reply:
Good point, comment added in https://github.com/discord/manifold/pull/13/commits/9e49cb69fb5a75c34638c04ef60ed5080efa769e